components/output/index.rst

```diff
@@ -38,9 +39,14 @@ Configuration variables:
   automatically be switched on too.
 - **inverted** (*Optional*, boolean): If the output should be treated
   as inverted. Defaults to ``False``.
+- **min_power** (*Optional*, float): Only for float outputs. Sets the
+  minimum output value of this output platform. Each value will be
```

Comment:
I think we can drop the comment about how the value will be calculated. I think it's pretty clear from the names what it does and the comment just makes things harder to understand.

Reply:
Sure, I agree that's a bit redundant, I've cleaned it up. Thanks.